src/server/project.ts

```diff
@@ -1034,7 +1042,15 @@ namespace ts.server {
                 };
 
                 const pluginModule = pluginModuleFactory({ typescript: ts });
-                this.languageService = pluginModule.create(info);
+                const newLS = pluginModule.create(info);
+                for (const k of Object.keys(this.languageService)) {
+                    if (!(k in newLS)) {
+                        this.projectService.logger.info(`Plugin activation warning: Missing proxied method ${k} in created LS. Patching.`);
+                        (newLS as any)[k] = (this.languageService as any)[k];

```

Comment:
Does this mean inside the language service methods that are being put directly onto the proxy, `this` will now refer to the proxy rather than language service instance?